delete_nonreduced_fuzz_inputs.sh

```diff
@@ -20,7 +20,13 @@ apt install -y \
   git \
   build-essential libtool autotools-dev automake pkg-config bsdmainutils python3 \
   libsqlite3-dev libevent-dev libboost-dev \
-  clang llvm
+  lsb-release wget software-properties-common gnupg
+
+wget https://apt.llvm.org/llvm.sh && chmod +x ./llvm.sh
+./llvm.sh 18 all
```

Comment:
I used the "ubuntu:latest" docker image, which still had clang 14. https://hub.docker.com/_/ubuntu says that should be `noble` but I just checked and on my machine this turned out to be `jammy`.

Happy to drop the commit, but this would also more easily let us bump the version to whatever we want.